playbooks/conferences.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Conferences
+description: Attending or speaking at a conference
+---
+
+# Conferences
+
+We encourage engineers to attend and speak at conferences. It is a great way to stay up-to-date with the industry,
+be inspired by challenges others are facing, develop personal career, and connect with the community. Here are some
+guidelines:
+
+## Attending Conferecens
+
+- We aim for each engineer to be able to attend a conference every year, subject to budget. The conference should
+  be related to individual's role or Artsy's challenges/technologies.
+- Please be mindful when expensing your travel. A good rule of thumb is to spend the money like your own. Refer to
+  [Atlas][travel-and-accommodations-on-atlas] for more details and rate card.
+- It's strongly recommended writing up take-aways and sharing with the team upon return, e.g. via [Lunch and
+  Learn][lunch-and-learn], [Show and Tell][show-and-tell], or a blog post.
```

Comment:
I’d like to see another option that may be a little less daunting for people than speaking/blogging. E.g. Slacking some notes (in a public channel) with links to projects/docs that were discussed during a talk and @ some people that you know are dealing with that space is a good ROI as well imo.

Reply:
Yeah good point! This should not be intimidating or discouraging.

Reply:
I agree that it can be lightweight! A list of take-aways and links to the best of the online content (decks/videos) could be enough, if shared with the most relevasnt distribution list of slack channel.